packages/entities/entities-routes/src/components/RouteList.vue

```diff
@@ -483,14 +483,19 @@ const confirmDelete = async (): Promise<void> => {
  */
 watch(fetcherState, (state) => {
   if (state.status === FetcherStatus.Error) {
-    errorMessage.value = t('errors.general')
+    errorMessage.value = {
+      title: t('errors.general'),
+    }
+    if (state.error?.response?.data?.message) {
+      errorMessage.value.message = state.error.response.data.message
+    }
```

Comment:
question here, but applies to all instances: are there times we would not want to surface the error message from the API to the user? Isn't that the point of the more generic messaging?

How does a downstream component know whether to display the `errorMessage.value.title` or the `errorMessage.value.message`? How to know which is appropriate?

Reply:
(Just) Some ideas for better UX: Maybe we can try to fold the "technical information" and show a general error message by default sometime in the future when we decide not to surface the detailed error. 

_Context: I'm recently fixing the OpenID Connect feature and playing with Okta_

![image](https://github.com/Kong/public-ui-components/assets/5277268/e15591c5-512e-46fd-9f92-9652ca4e7d4f)

Reply:
@adamdehaven Good question.
My opinion is, for the Kong Manager, we should always display the message returned by server. Since KM is designed as the frontend of the Admin API, anything returned from Admin API need to be present to the user.

For Konnect, it could be more complicated. I agree with @sumimakito, we can always use pre-defined phrase for these scenarios and hide server response in "technical details" for better user experience